src/components/CompareResults/SubtestsResults/SubtestsRevisionRow.tsx

```diff
@@ -77,21 +77,22 @@ function getStyles(themeMode: string) {
           padding: '4px 10px',
         },
         '.status-hint-improvement': {
-          backgroundColor: '#D8EEDC',
+          backgroundColor: '#004725',
+          marginTop: '2px',
         },
         '.status-hint-regression': {
-          backgroundColor: '#FFE8E8',
+          backgroundColor: '#690F22',
```

Comment:
Moving forward, we want to use MUI theme to update the styles. So, for this PR could you please delete the backgroundColor styles for both improvement and regression  and add the `color` prop to the icons and have it equal to `success`

```
 {improvement ? <ThumbUpIcon color='success' /> : null}
{regression ? <ThumbDownIcon color='success' /> : null}
```

Reply:
Should this also be done for `RevisionRows` or only for SubtestsRevisionRow?

Reply:
you can do it for both

Reply:
I believe carla wanted to say `error` for the `regression` line.

@Carla-Moz I think this works for the icons but I don't think we have anything for the background color yet. I believe we need to add `success` and `error` properties to the theme in https://github.com/mozilla/perfcompare/blob/main/src/theme/protocolTheme.ts#L12 and https://github.com/mozilla/perfcompare/blob/main/src/theme/protocolTheme.ts#L34 ?

Reply:
> you can do it for both

Okay noted

Reply:
> I believe carla wanted to say `error` for the `regression` line.
> 
> 
> 
> @Carla-Moz I think this works for the icons but I don't think we have anything for the background color yet. I believe we need to add `success` and `error` properties to the theme in https://github.com/mozilla/perfcompare/blob/main/src/theme/protocolTheme.ts#L12 and https://github.com/mozilla/perfcompare/blob/main/src/theme/protocolTheme.ts#L34 ?

Yes that's why I told Neta to add the new properties to the protocolTheme file in the next commit. Please check my comment further below.